Document/0x05b-Basic-Security_Testing.md

```diff
@@ -408,46 +325,11 @@ $ drozer console connect
 
     `dz> run app.broadcast.send --action (broadcast receiver name) -- extra (number of arguments)`
 
-###### Using Modules
-
-Out of the box, Drozer provides modules for investigating various aspects of the Android platform and a few remote exploits. You can extend Drozer's functionality by downloading and installing additional modules.
-
-###### Finding Modules
-
-The official Drozer module repository is hosted alongside the main project on GitHub. This is automatically set up in your copy of Drozer. You can search for modules with the `module` command:
-
-```shell
-dz> module search tool
-kernelerror.tools.misc.installcert
-metall0id.tools.setup.nmap
-mwrlabs.tools.setup.sqlite3
-```
-
-For more information about a module, pass the `–d` option to view the module's description:
-
-```shell
-dz> module  search url -d
-mwrlabs.urls
-    Finds URLs with the HTTP or HTTPS schemes by searching the strings
-    inside APK files.
-
-        You can, for instance, use this for finding API servers, C&C
-    servers within malicious APKs and checking for presence of advertising
-    networks.
-
-```
-
-###### Installing Modules
-
-You can install modules with the `module` command:
-
-```shell
-dz> module install mwrlabs.tools.setup.sqlite3
-Processing mwrlabs.tools.setup.sqlite3... Already Installed.
-Successfully installed 1 modules, 0 already installed
-```
-
-This will install any module that matches your query. Newly installed modules are dynamically loaded into the console and are available immediately.
+Learn more about drozer's security assessment and exploitation features by checking the following resources:
+- [Command Reference](https://mobiletools.mwrinfosecurity.com/Command-Reference/ "drozer's Command Reference")
+- [Using drozer for application security assessments](https://mobiletools.mwrinfosecurity.com/Using-Drozer-for-application-security-assessments/ "Using drozer for application security assessments")
+- [Exploitation features in drozer](https://mobiletools.mwrinfosecurity.com/Exploitation-features-in-drozer/ "Exploitation features in drozer")
+- [Using modules](https://mobiletools.mwrinfosecurity.com/Installing-modules/)
```

Comment:
what i missed when diving into drozer, was a basic architecture description of how drozer is divided and how that all comes togeter. Maybe it's good to give modules a bit more detail
Next, the Sieve application they refer to is pretty old, would it be a good idea to refer to one of the other tutorials using drozer on another vulnerable app?

Reply:
To speed up with the overall tickets and finish the restructuring I'd put that on a separate ticket. It is now clear what is to be done:
- basic architecture
- modules
- review Sieve
- talk about the highlights: why do we want to use drozer, there's some things you can actually do via adb, or some grepping, without having to install drozer only for that. But for other things it is really helpful.

Reply:
yes, create a separate ticket , that's a good idea

Reply:
done, now in #1270